app/components/ui/forms.tsx

```diff
@@ -3,6 +3,44 @@ import { Link } from '@remix-run/react';
 
 import clsx from 'clsx';
 
+export const MAX_FILE_SIZE_MB = 8;
+export const ACCEPTED_IMAGE_TYPES = ['image/jpeg', 'image/jpg', 'image/png', 'image/webp'];
+
+type ImageUploadProps = InputHTMLAttributes<HTMLInputElement> & {
+  label: ReactNode;
+  centerText?: boolean;
+  showLabel?: boolean;
+  maxFileSizeMB?: number;
+  fileAccept?: string;
+};
+export function ImageUpload({
+  label,
+  fileAccept = 'image/*',
```

Comment:
Should we allow all image types or would it be better to use `ACCEPTED_IMAGE_TYPES` here so we limit the file types to ones that would supported by more browsers?